qt/python/mantidqt/gui_helper.py

```diff
@@ -19,6 +19,8 @@ def set_matplotlib_backend():
     the name of the backend to be the name to be used for importing the
     correct matplotlib widgets.'''
     backend = matplotlib.get_backend()
+    if 'MPLBACKEND' in os.environ:
```

Comment:
Looking at the test failures on Windows (I'm not sure why the other OSes have passed) then I think we will have to remove this `if`. The fix below will ensure the system tests are okay but `set_matplotlib_backend` is used in a way that means making the other tests work is a bit fiddly.

Reply:
That's been removed. 

Removing the import in __init__ is sufficient to stop producing the warning